eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -38,7 +38,9 @@ param(
 
   [Parameter(Mandatory = $true)]
   $PRTitle,
-  $PRBody = $PRTitle
+
+  [Parameter(Mandatory = $true)]
+  $PRBody
```

Comment:
Why make this mandatory? If it isn't specified we should just default it to PRTitle like it was before.